osler/workup/models.py

```diff
@@ -137,6 +75,7 @@ class AbstractBasicNote(Note):
 
     title = models.CharField(max_length=200)
     text = models.TextField()
+    encounter = models.ForeignKey(Encounter, on_delete=models.CASCADE)
```

Comment:
putting a foreign key in an abstract model creates problems when you want to get encounter.basicnote_set later. move this field to the child classes

Reply:
Actually, are we sure this is correct? It seems like a pretty bad DRY violation, right?

Reply:
you are right. forget when I had an issue with that, but it shouldn't apply here